Configure

```diff
@@ -4865,18 +4865,21 @@ esac
 case "$ccname" in
     gcc)
 	$echo 'extern int foo;' > try.c
-	set X `$cppstdin -v try.c 2>&1 | $awk '/^#include </,/^End of search /'|$cppfilter $grep '/include'`
+	set X `$cppstdin -v try.c 2>&1 | $awk '/^#include </,/^End of search /'|$cppfilter $grep '/include$'`
 	shift
 	if $test $# -gt 0; then
 	    incpth="$incpth $*"
 	    incpth="`$echo $incpth|$sed 's/^ //'`"
-	    for i in $*; do
-		j="`$echo $i|$sed 's,/include$,/lib,'`"
-		if $test -d $j; then
-		    libpth="$libpth $j"
-		fi
-	    done
-	    libpth="`$echo $libpth|$sed 's/^ //'`"
+	   for i in $*; do
+	       j="`$echo $i|$sed 's,/include$,/lib,'`"
+	       if $test -d $j; then
+		   case " $libpth " in
+		   *" $j "*) ;;
+		   *) libpth="$libpth $j";;
+		   esac
+	       fi
+	   done
+	   libpth="`$echo $libpth|$sed 's/^ //'`"
```

Comment:
Did the leading whitespace also change? I find this a weird diff and hard to spot the diff and it also looks like the new indentation is wrong

If `include-fixed` is still present here, it should be catered for? (as here the `/include$` is anchored

Reply:
> Did the leading whitespace also change? I find this a weird diff and hard to spot the diff and it also looks like the new indentation is wrong

Yeah, fixed that in the other commits, forgot about this one.

> If include-fixed is still present here, it should be catered for? (as here the /include$ is anchored

Yeah, that must be handled somehow, though I'm also not entirely fond of how that's done here.